spec/factories/ext_management_system.rb

```diff
@@ -28,6 +28,12 @@
         create_list :storage, evaluator.storage_count, :ext_management_system => ems
       end
     end
+
+    trait :with_invalid_authentication do
+      after(:create) do |x|
+        x.authentications << FactoryGirl.create(:authentication, :resource => x, :status => "invalid")
```

Comment:
When you use the `<<` operator on an association, you shouldn't need the `:resource => x` in `create`